pkg/skr/cloudresources/deleteUiCms.go

```diff
@@ -0,0 +1,43 @@
+package cloudresources
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+)
+
+func deleteUiCms(ctx context.Context, st composed.State) (error, context.Context) {
+
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+
+	uiCmList := util.NewUiCmListUnstructured()
```

Comment:
Use typed k8s.io/api/core/v1 ConfigMap

```suggestion
	uiCmList := &corev1.ConfigMapList{}
```